### PR TITLE
Added blueprint check

### PR DESCRIPTION
**Pull request by dafortin**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Added** blueprint check

### CODE REVIEW

File: blueprints/ember-frost-sort/index.js [hunk 1]
```diff
@@ -1,22 +1,33 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-computed-decorators', target: '~0.2.2'},
```
Comment (rox163) [inline blueprints/ember-frost-sort/index.js]: can we move to 0.3.x for this addon?

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-sort/pull-request/60/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-sort)

File: blueprints/ember-frost-sort/index.js [hunk 2]
```diff
@@ -1,22 +1,33 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-computed-decorators', target: '~0.3.0'},
+      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-elsewhere', target: '~0.4.1'},
```
Comment (sglanzer-fire) [inline blueprints/ember-frost-sort/index.js]: Does `sort` use `elsewhere` or is that just a factor of using the select?

File: blueprints/ember-frost-sort/index.js [hunk 3]
```diff
@@ -1,22 +1,33 @@
...
+      {name: 'ember-hook', target: '^1.4.1'},
+      {name: 'ember-prop-types', target: '^3.10.2'}
```
Comment (sglanzer-fire) [inline blueprints/ember-frost-sort/index.js]: Will `hook`, `prop-types` and `computed-decorators` come in from `core`?

**Review by sglanzer-fire — CHANGES_REQUESTED** [review]
I'd just like confirmation if the nested installation from `core` will cover the other addons or if the need to be explicit

File: blueprints/ember-frost-sort/index.js (hunk 2 above)
Comment (dafortin) [reply blueprints/ember-frost-sort/index.js]: no I will remove it from here.

File: blueprints/ember-frost-sort/index.js (hunk 3 above)
Comment (dafortin) [reply blueprints/ember-frost-sort/index.js]: They will, but this addon is also using those so I'm expecting that we need those here.

Comment (sglanzer-fire) [reply blueprints/ember-frost-sort/index.js]: If core handles the installation then I believe you'll only need `core` - the hook here will only determine installation, not consumption.  Consumption would be a factor of the resolver and `ember-cli` registering the addon, which should happen automagically if the packages are present

Give it a shot, but I'm pretty sure I'm correct about this

Comment (dafortin) [reply blueprints/ember-frost-sort/index.js]: Now I'm curious :P I will try it for sure :)

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-sort/pull-request/60/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-sort)

**Review by sglanzer-fire — APPROVED** [review]

**Pull request merged**